commands/command_migrate_export.go

```diff
@@ -122,8 +150,19 @@ func migrateExportCommand(cmd *cobra.Command, args []string) {
 		ExitWithError(err)
 	}
 
-	// Only perform `git-checkout(1) -f` if the repository is
-	// non-bare.
+	// Prune the cache
+	for _, object := range exported {
```

Comment:
Will `exported` ever contain an object evicted by the migration, but used elsewhere on disk? An example demonstrating this is a repository with two files, both tracked in LFS, sharing the same contents at different paths. If only one of those paths is migrated out, the other will still wish to retain the object.

Perhaps we could invoke the `git lfs prune` command here, instead?